jsapp/js/api.endpoints.ts

```diff
@@ -1,4 +1,5 @@
 export const endpoints = {
+  ASSET_HISTORY: '/api/v2/assets/:asset_id/history/',
```

Comment:
```suggestion
  ASSET_HISTORY: '/api/v2/assets/:asset_uid/history/',
```
I don't think we have such thing as asset **id** :)